### PR TITLE
Create args-print

### PR DESCRIPTION
**Pull request by MateuszSab**
Mateusz project

### CODE REVIEW

**Review by jaceklaskowski — CHANGES_REQUESTED** [review]
1. Use `src/main/scala` for the app
2. Where are the other files (e.g. `build.sbt`, `project.properties`)?

**Comment by MateuszSab** [comment]
fixed, I hope

**Review by jaceklaskowski — CHANGES_REQUESTED** [review]
1. `project/target` to be deleted
2. `project/project/target/` to be deleted too
3. `README.md` missing?

**Comment by MateuszSab** [comment]
what is wrong this time?

**Comment by MateuszSab** [comment]
Czy teraz jest w porządku?

File: mateusz/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+My scopt project
```
Comment (jaceklaskowski) [inline mateusz/README.md:1]: 1. No title?!
2. How to build and run?

**Review by jaceklaskowski — CHANGES_REQUESTED** [review]
1. All the files should go under `docker-scopt`

**Comment by MateuszSab** [comment]
files refactored, readme updated

**Pull request merged**